webpack/assets/javascripts/react_app/routes/HostWizard/index.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import { foremanUrl } from '../../common/urlHelpers';
 import HostWizard from './HostWizardPage';
 
 export default {
-  path: '/host_wizard',
+  path: foremanUrl('/host_wizard'),
```

Comment:
Isn't it redundant?

Reply:
I don't get your point :thinking: can we discuss offline? I think you are asking about similar thing, that's explained ^^